.github/workflows/pr-title-validation.yaml

```diff
@@ -17,6 +17,9 @@ name: "PR Title Checker"
 permissions:
   contents: read
 
+  # Needed so pr-title-checker can post a comment or set a PR status when title doesn't match rules
+  pull-requests: write
```

Comment:
nitpicking - at some places comment is above, at some it is within the same line